index.html

```diff
@@ -1012,18 +1068,17 @@ <h3><dfn data-lt="GpgVerificationKey2020" data-dfn-type="dfn" id="GpgVerificatio
 }
       </pre>
     </section>
-  </section>
 </section>
 
   <section>
-    <h1>The Parameters Extension Registry</h1>
+    <h1>Remote Parameters Extensions</h1>
```

Comment:
Maybe third-party rather than remote?

Reply:
We might consider calling them "Working Group extensions" and "Third party extensions"

Reply:
Third party seems like it might be better.